src/repositories/invoice-repository.ts

```diff
@@ -103,6 +103,11 @@ export class InvoiceRepository implements IInvoiceRepository {
         always(undefined),
         prop('createdAt'),
       ),
+      verify_url: ifElse(
+        propSatisfies(isNil, 'verifyURL'),
+        always(undefined),
+        prop('verifyURL'),
+      ),
```

Comment:
```suggestion
      verify_url: prop('verifyURL'),
```
essentially the same :)

Reply:
Should I change the above ones as well along with this?

Reply:
oh yeah, deja vu! because `isNil` checks if the value is undefined, so the check is not needed